ui2/src/pages/landing/Landing.tsx

```diff
@@ -90,21 +112,28 @@ const ProfileSection: React.FC<{ action: () => void }> = ({ action }) => {
   const { contracts: identities, loading: identitiesLoading } = useStreamQueries(VerifiedIdentity);
   const partyIdentity = identities.find(id => id.payload.customer === customer);
 
+  const damlHubParty =
+    isDamlHubParty(customer) && name !== customer ? (
+      <p>
```

Comment:
would it also make sense to put the `<p>` on line 46 rather than potentially wrapping an `input` in a `<p>` (if its a daml hub party)

Reply:
Thanks, yeah that makes more sense. I also realized that since I'm checking `isDamlHubParty` out here (line 115), I don't need to do the same thing inside the component itself